.github/workflows/build_librealsense_ROS2_Ubuntu.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build_ROS2
```

Comment:
Lets change the file name:
`[build_librealsense_ROS2_Ubuntu.yaml]`
to
`build-ROS2-package-CI`

Reply:
The file was renamed.
Thank you for the good suggestion.